addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.js

```diff
@@ -27,6 +27,12 @@ export class AvatarCardPopover extends Component {
             id: this.props.id,
             model: this.props.model,
         });
+        onWillUnmount(() => {
+            if (this.partner?.stopRealtimeTzDiff) {
+                this.partner.stopRealtimeTzDiff();
+                this.partner.stopRealtimeTzDiff = null;
+            }
+        });
```

Comment:
This part is strange, it doesn't really make sense for the stop to be in a different component than the one starting it.